dist/tmpfiles.d/zincati.conf

```diff
@@ -12,3 +12,10 @@ d     /run/zincati/public    0775 zincati zincati  -   -
 
 # Legacy symlink to metrics socket
 L+    /run/zincati/private/metrics.promsock  - - - - ../public/metrics.promsock
+
+# Runtime MOTD messages which Zincati can write to
+d     /run/zincati/public/motd.d  0755 zincati zincati  -   -
```

Comment:
I think this was `0775` in the previous iteration. What's the reason for changing it?

Reply:
Sorry, had been experimenting (seeing if `0664` worked), and typed the wrong value when fixing it back. Thanks for catching, fixed!